gcp_variant_transforms/libs/bigquery_util.py

```diff
@@ -156,3 +177,15 @@ def get_vcf_num_from_bigquery_schema(bigquery_mode, bigquery_type):
 def get_supported_bigquery_schema_types():
   """Returns the supported BigQuery field types."""
   return [item.value for item in _SupportedTableFieldType]
+
+
+def get_avro_type_from_bigquery_type_mode(bigquery_type, bigquery_mode):
+  # type: (str, str) -> Union[str, List[str, str]]
+  if not bigquery_type in _BIG_QUERY_TYPE_TO_AVRO_TYPE_MAP:
+    raise ValueError('Unknown Avro equivalent for type {}'.format(
+        bigquery_type))
+  t = _BIG_QUERY_TYPE_TO_AVRO_TYPE_MAP[bigquery_type]
```

Comment:
nit: in general, we've tried to avoid using short form variables (e.g. `t`, `f`, etc) in our code as it makes it harder to read (the only exception has been for one-line for loops). Consider renaming these to `avro_type` etc.